reader/src/test/java/org/kaazing/monitoring/reader/CollectorFactoryTest.java

```diff
@@ -18,33 +18,21 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
 package org.kaazing.monitoring.reader;
 
-import org.kaazing.monitoring.reader.api.MetricsCollector;
-import org.kaazing.monitoring.reader.impl.MetricsCollectorAgrona;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-/**
- * Factory class that returns a MetricsCollector instance
- */
-public class MetricsCollectorFactory {
+import org.junit.Test;
+import org.kaazing.monitoring.reader.exception.MetricsReaderException;
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(MetricsCollectorFactory.class);
+public class CollectorFactoryTest {
 
-    /**
-     * Initializes a MetricCollector instance
-     * @return MetricsCollectorAgrona
-     */
-    public static MetricsCollector getInstance() {
-        Configuration config = new Configuration();
-
-        if (!config.loadConfigFile()) {
-            LOGGER.error("There was a problem with the configuration file. Exiting application.");
-            return null;
+    @Test
+    public void testCollectorFactory() throws Exception {
```

Comment:
Any other tests for this factory other than making sure it can get initialized?  Anything else you think needs tested related to this functionality?